reflex/event.py

```diff
@@ -612,6 +612,34 @@ def remove_local_storage(key: str) -> EventSpec:
     )
 
 
+def clear_session_storage() -> EventSpec:
+    """Set a value in the session storage on the frontend.
```

Comment:
should be "clear all values", looks like we missed this in the local_storage also